src/services/futures/trade.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from "../..";
+import { FUTURES_TRACKING, PRICE_IMPACT_DELTA } from "./constants";
+import ISynthetixFuturesMarketV2 from "../../abi/ISynthetixFuturesMarketV2.json";
+
+export function getFuturesChangePositionTxData(
+  amount: ethers.BigNumber | string
+): string {
+  return new ethers.utils.Interface(
+    ISynthetixFuturesMarketV2.abi
+  ).encodeFunctionData("modifyPositionWithTracking", [
```

Comment:
To make sure we pay low fees on Perps v2, we should use `submitOffchainDelayedOrderWithTracking` function.
This will then be only a 0.05 - 0.1% trading fee.
It is however a little more complicated:
- a keeper executes the trade 10-15sec later
- need to include a keeper fee for the gas

FYI they achieve the low trading fee because the keeper gets the oracle price for the asset and executes the trade (lower risk of frontrunning).
![image](https://user-images.githubusercontent.com/44114383/217993320-39f18f18-9d5e-4f5a-a5b2-64f2ec804f5a.png)